crates/orderbook/openapi.yml

```
@@ -345,39 +357,40 @@ paths:
           description: No competition information available for this auction id.
   /api/v1/solver_competition/by_tx_hash/{tx_hash}:
     get:
-      summary: Information about solver competition
+      summary: Get information about solver competition.
       description: |
-        Returns the competition information by transaction hash.
+        Returns the competition information by `tx_hash`.
       parameters:
         - name: tx_hash
+          description: Transaction hash in which the competition was settled.
           in: path
           required: true
           schema:
             $ref: "#/components/schemas/TransactionHash"
       responses:
         200:
-          description: competition info
+          description: Competition
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/SolverCompetitionResponse"
         404:
-          description: No competition information available for this tx hash.
+          description: No competition information available for this `tx_hash`.
   /api/v1/version:
     get:
-      summary: Information about the current deployed version of the API
+      summary: Get the API's current deployed version.
       description: |
         Returns the git commit hash, branch name and release tag (code: https://github.com/cowprotocol/services).
       responses:
         200:
-          description: version info
+          description: Version
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/VersionResponse"
   /api/v1/app_data/{app_data_hash}:
     get:
-      summary: Retrieve full app data from contract app data hash.
+      summary: Get the full appData from contract appDataHash.
```

Should you `\`` for consistency?

```suggestion
      summary: Get the full `appData` from contract `appDataHash`.
```

---

Nice catch, by this time it started to all blur together 🫨